docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -199,8 +243,82 @@ Create the required StatefulSet using:
 You can check its status using the commands ``kubectl get statefulsets -w``
 and ``kubectl get svc -w``
 
- 
-Step 6: Run BigchainDB as a Deployment
+You may have to wait for upto 10 minutes wait for disk to be created
+and attached on the first run. The pod can fail several times with the message
+specifying that the timeout for mounting the disk has exceeded.
+
+
+Step 7: Initialize a MongoDB Replica Set - Optional
+---------------------------------------------------
+
+This step is required only if you are planning to set up multiple
+`BigchainDB nodes
+<https://docs.bigchaindb.com/en/latest/terminology.html#node>`_, else you can
+skip to the :ref:`step 9 <Step 9: Run BigchainDB as a Deployment>`.
+
+
+Login to the running MongoDB instance and access the mongo shell using:
+
+.. code:: bash
+   
+   $ kubectl exec -it mdb-0 -c mongodb -- /bin/bash
+   $ mongo --port 27017
+
+We initialize the replica set by using the ``rs.initialize()`` command from the
+mongo shell. Its syntax is:
+
+.. code:: bash
+
+    rs.initiate({ 
+        _id : "<replica-set-name",
+        members: [ { 
+          _id : 0,
+          host : "<fqdn of this instance>:<port number>"
+        } ]
+    })
+
+An example init command might look like:
+
+.. code:: bash
+   
+   > rs.initiate({ _id : "bigchain-rs", members: [ { _id : 0, host : "bdb-cluster-0.westeurope.cloudapp.azure.com:27017" } ] })
```

Comment:
What should they put for the `<fqdn of this instance>`? The same value used for `data.fqdn` in `mongo-cm.yaml`? But the example seems to be the FQDN of a cluster... but a cluster isn't an instance... please add more instructions. And if the FQDN is supposed to be for the "cluster," maybe say how to determine that. (In Azure Portal, is it the "Master FQDN" when they look at the Overview for their ACS cluster resource? If so, why call that the "`<fqdn of this instance>`"?)

Also, why is the port 27017 in the example? That's not the usual MongoDB port. Can they just pick a random valid port, provided it's not one that's going to be used by something else?